TipCatDotNet.ApiTests/AccountServiceTests.cs

```diff
@@ -93,12 +99,29 @@ public async Task Add_should_add_account()
         }
 
 
+        [Fact]
+        public async Task Add_should_create_default_facility()
+        {
+            const string expectedFacilityName = "Default facility";
```

Comment:
This one is quite strange, because it tests not our code, but mostly framework logic